applications/pfem_2_application/custom_utilities/move_particle_utility_pfem2.h

```diff
@@ -2742,7 +2743,7 @@ namespace Kratos
 				bool is_found_2 = CalculatePosition(geom,coords[0],coords[1],coords[2],N);
 				if (is_found_2)
 				{
-					pelement=neighb_elems(i).lock();
+					pelement = neighb_elems[i].shared_from_this();
```

Comment:
@RiccardoRossi shouldn't there be a better solution to this?

I mean to make it work again it is ok, but to me it seems like alos some things should be rewrtitten to use intrusive_ptr (or even raw ptrs) instead of shared_ptr in the elements right?
Ther the `shared_from_this` would not be needed I think